easybuild/easyconfigs/h/hwloc/hwloc-1.11.12-GCCcore-9.3.0.eb

```diff
@@ -0,0 +1,48 @@
+easyblock = 'ConfigureMake'
+
+name = 'hwloc'
+version = '1.11.12'
```

Comment:
@terjekv This one means trouble, since we already have an `hwloc` as a part of `foss/2020a` (as a dep for OpenMPI).

Is `hwloc` 2.2.0 (via ` hwloc-2.2.0-GCCcore-9.3.0.eb`) not an option?

Reply:
Oh, I thought we treated `hwloc-1*` and `hwlock-2` as different projects. I can test `hwloc-2.2.0-GCCcore-9.3.0.eb`

Reply:
That is even what TF expects :)

Reply:
Tested with `hwloc-2.2.0-GCCcore-9.3.0.eb`. Seems ok. 👍